presto-product-tests/src/main/java/com/facebook/presto/tests/cassandra/DataTypesTableDefinition.java

```diff
@@ -50,11 +50,17 @@ private DataTypesTableDefinition() {}
         RelationalDataSource dataSource = () -> {
             try {
                 return ImmutableList.<List<Object>>of(
-                        ImmutableList.of("abc", 123L, Bytes.fromHexString("0x48495051"), false, new BigDecimal(456.789), 1234.5678,
+                        ImmutableList.of("abc", 123L, Bytes.fromHexString("0x30313233"), false, new BigDecimal(456.789), 1234.5678,
                                 90123.456f, ImmutableSet.of(1, 2, 3), Inet4Address.getByName("127.0.0.1"), 1, ImmutableList.of(4, 5, 6), ImmutableMap.of("a", 1, "b", 2),
                                 ImmutableSet.of(4, 5, 6), "this is a text value", Timestamp.valueOf("2011-02-03 04:05:00"),
                                 UUID.fromString("d2177dd0-eaa2-11de-a572-001b779c76e3"), UUID.fromString("01234567-0123-0123-0123-0123456789ab"),
                                 "abc", new BigInteger("567")),
+                        ImmutableList.of("the quick brown fox jumped over the lazy dog", 9223372036854775807L, Bytes.fromHexString("0x3031323334"),
+                                false, BigDecimal.valueOf(Double.MAX_VALUE), Double.MAX_VALUE,
```

Comment:
A couple of things:
* Try the highest possible Presto decimal (precision 38, all 9's). It wouldn't be a bad idea to try both long and short decimal from Presto.
* non-ascii characters for varchar and text (and other text-ish fields)
* Long.MAX_VALUE instead of the value
* true instead of false (just to test both)
* Try Bytes.fromHexString("0x00") to see if null is handled properly